src/amo/components/Addon/index.js

```diff
@@ -308,6 +311,15 @@ export class AddonBase extends React.Component {
             <h1 className="Addon-title" {...titleProps} />
             <p className="Addon-summary" {...summaryProps} />
 
+            <div className="Addon-badges">
+              {isFeatured ? (
+                <Badge
+                  type="featured"
+                  label={i18n.sprintf(i18n.gettext('Featured %(addonType)s'), { addonType })}
```

Comment:
So in fact, after discussing #2979, we should just have a switch case for this translation for all the add-on types.

We tend to only feature extensions and themes though, so I think the switch/case can just be for `ADDON_TYPE_EXTENSION`, `ADDON_TYPE_THEME`, and then a default with "add-on".